cardano-cli/test/cli/core/common

```diff
@@ -99,7 +99,7 @@ report_result () {
 
 setup_data_dir () {
   DATA_DIR="test/cli/${1}/data"
-  if test -d cardano-cli ; then
+  if test ! -f cardano-cli.cabal ; then
```

Comment:
More reliable test to avoid the case where accidentally created sub-directory called `cardano-cli` breaks the setup function.